bin/bootstrax

```diff
@@ -1069,7 +1069,6 @@ def upload_file_metadata(rd):
                     chunk_mb = [chunk['nbytes'] / (1e6) for chunk in md['chunks']]
                     data_size_mb = int(np.sum(chunk_mb))
                     avg_data_size_mb = int(np.average(chunk_mb))
-                    lineage_hash = md['lineage_hash']
```

Comment:
I think we want to leave this in right? Unless it is already included somehow?

Reply:
I think this was resolved 👍